fast_api_als/database/db_helper.py

```diff
@@ -25,6 +26,7 @@ def __init__(self, session: boto3.session.Session):
         self.geo_data_manager = self.get_geo_data_manager()
         self.dealer_table = self.ddb_resource.Table(constants.DEALER_DB_TABLE)
         self.get_api_key_author("Initialize_Connection")
+        info("session is created")
```

Comment:
logger.info() would look much better when someone is going through your code. Right now it's confusing whether this is an info function of the logger, or you've implemented the info() function somewhere.

Reply:
Quick tip, search in google how you can get logger from logging library.
and then use logger.info() or logger.error()